src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -367,6 +367,17 @@ public function get_items( $request ) {
 			add_filter( 'post_password_required', array( $this, 'check_password_required' ), 10, 2 );
 		}
 
+		$parent_ids = array();
+		foreach ( $query_result as $post ) {
+			if ( $post->post_parent > 0 ) {
+				$parent_ids[] = $post->post_parent;
+			}
+		}
+
+		if ( ! empty( $parent_ids ) ) {
+			_prime_post_caches( $parent_ids, false );
+		}
```

Comment:
`_prime_post_caches` can handle empty arrays so removing the `!empty` sounds safe to me.

```suggestion
		_prime_post_caches( $parent_ids, false );
```

Reply:
Why do a function when it is not needed. Take CPU cycles that are not needed.